docs(createAutocomplete): rewrite docs (#472)
algolia/autocomplete: packages/website/docs/partials/createAutocomplete-props.md

> `string` | defaults to `"autocomplete-0"` (incremented for each instance)

> An ID for the autocomplete. This is necessary to create accessible attributes.

Reviewer: "This is necessary" sounds like it is required, but it's not.

```suggestion
An ID for the autocomplete to use accessible attributes.
```
